### DISCUSSION TITLE
Zeek Intel

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#5932] [Unsupported Versions] [answered]
Hello everyone!

Anyone have any luck using these intel files for zeek intelligence framework in Security Onion 2? 

I'm assuming since the syntax is correct,  we could probably just save each individual file with the .dat extension and copy to the correct folder for salt to push to the forward sensor? Do they all need to be in one file or could we have multiple .dat files for zeek to look at?

https://github.com/CriticalPathSecurity/Zeek-Intelligence-Feeds

**Comment by dougburks** [comment, marked as answer]
I think there are probably a few different ways to do this:
- follow https://docs.securityonion.net/en/2.3/zeek.html#intel and combine all intel into the default `intel.dat`
- follow https://docs.securityonion.net/en/2.3/zeek.html#intel but override the default `intel/__load__.zeek` to load all of the individual dat files
- follow https://docs.securityonion.net/en/2.3/zeek.html#custom-scripts since https://github.com/CriticalPathSecurity/Zeek-Intelligence-Feeds already has its own `__load__.zeek` referencing its individual dat files

**Comment by ColeVan** [comment]
Copying the intel files from https://github.com/CriticalPathSecurity/Zeek-Intelligence-Feeds and appending the _load_.zeek as seen below allowed me to successfully test nslookup to one cobaltstrike domain from a windows hosts. (Beware! Test at your own risk) 

Hopefully this helps anyone out in the future for wanting to use these zeek intelligence feeds. Some intel files had false positives which are not hard to spot.
![image](https://user-images.githubusercontent.com/70167373/138003617-4fc354e0-a237-44c8-9709-6bea06d996fb.png)
![image](https://user-images.githubusercontent.com/70167373/138005862-f75661b5-a4f1-4d20-8f8c-3976a64b3550.png)

**Reply by cyb3rslack3r** [reply]
I appreciate you posting this here.  I did not have successful results when trying to duplicate your steps.  The changes I made to the __load__.zeek file kills zeek and I notice the service reboots every 10 minutes.  I have tried different directories, but not sure what I am missing.  I know the security onion steps show using the path "/opt/so/saltstack/local/salt/zeek/policy/intel/", which I tried that too.  I know the path you have in your screenshot is not native to SO2 and needs to be created.  Do you have a __load__.zeek file in the "/opt/so/saltstack/local/salt/zeek/policy/intel/" path or only in the path above.  Thanks

**Reply by ColeVan** [reply]
Hello,

I replaced the original _load_.zeek file with the one at this link below. 

https://github.com/CriticalPathSecurity/Zeek-Intelligence-Feeds/blob/master/main.zeek

I obviously renamed it back to _load_.zeek and made sure all intel files in "/opt/so/saltstack/default/salt/zeek/policy/intel/" ended with .intel

I noticed once you sudo cp /opt/so/saltstack/default/salt/zeek/policy/intel/* /opt/so/saltstack/local/salt/zeek/policy/intel/  that zeek will error out if you have any files ending .dat extension. Ensure your mapping file "_load_.zeek" is correct and that you only have .intel files copied from "/opt/so/saltstack/default/salt/zeek/policy/intel/* " to "/opt/so/saltstack/local/salt/zeek/policy/intel/" or zeek will not be happy. Lastly, make sure to conduct a highstate to your minion and restart zeek. Hope this helps.
